sdk/core/core/inc/az_http_policy.h

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZ_HTTP_POLICY_H
+#define AZ_HTTP_POLICY_H
+
+#include <az_contract.h>
+#include <az_http_request_builder.h>
+#include <az_pair.h>
+#include <az_result.h>
+#include <az_span.h>
+#include <az_str.h>
```

Comment:
You only need az_mut_span.h, az_span.h, az_result.h, az_http_request_builder.h

Reply:
done